scripts/test_hive.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/bash
+
+# Wait until hive test server is ready, port 8899
+# is a status port indicates the hive server container
+# is ready, see .travis.yml for the details
+while [ 1 ]
+do
+curl http://localhost:8899 2>/dev/null
+if [ $? -eq 0 ]; then
+break
+else
+echo "still waiting, hive server is not ready..."
+sleep 5
+fi
+done
+
```

Comment:
Since we are switching back to `sqlflow/sqlflow:latest`, the following lines can be removed.
```
set -e
. /miniconda/etc/profile.d/conda.sh
source activate sqlflow-dev
```

Reply:
Thanks, will clean up in next PR.